docs-source/source/contribute.rst

```diff
@@ -29,8 +29,68 @@ guide focuses on the requirements to comply with it. Nevertheless, we accept any
 important and making our library more robust over time is of paramount importance. Please don't hesitate to submit a
 Github pull request for something as small as a typo.*
 
+Open source contributions
```

Comment:
This is a repeating a bit what is stated above. 
I think we should remove the list as it is almost the same as above. 
Cognite Hub is described in the paragraph above as well. 

I like that you added Code of Conduct